src/subscription_manager/managercli.py

```diff
@@ -2088,6 +2088,18 @@ def _do_command(self):
         if self.options.pool or self.options.file:
             self.auto_attach = False
 
+        # Do not try to do auto-attach, when simple content access mode is used
+        # BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1826300
+        if self.auto_attach is True:
+            if is_simple_content_access(uep=self.cp, identity=self.identity):
+                owner = get_current_owner(self.cp, self.identity)
+                owner_name = owner['displayName']
+                # We could also display owner ID: `owner_id = owner['key']` (not sure)
+                print(_('Ignoring request to auto-attach. '
+                        'It is disabled for organization: "%s" because of the content access mode setting.')
```

Comment:
Perhaps:
... because Simple Content Access is in use.

Reply:
Good point. I just copy pasted suggestion from BZ bug report.